stardog/templates/cronjob.yaml

```
@@ -68,16 +68,35 @@ spec:
             - -c
             - |
               set -e
-              BACKUP_TARGET="s3://${S3_ENDPOINT}/${S3_BUCKET_NAME}/${S3_PATH_PREFIX}/$(date --utc +%Y-%m-%d_%H%M)?AWS_ACCESS_KEY_ID=${S3_ACCESS_KEY}&AWS_SECRET_ACCESS_KEY=${S3_SECRET_KEY}&region=${S3_REGION}"
-              /opt/stardog/bin/stardog-admin db status --passwd "${STARDOG_ADMIN_PW}" {{ $db.name | quote }}
-              /opt/stardog/bin/stardog-admin db backup --passwd "${STARDOG_ADMIN_PW}" \
-                --to "${BACKUP_TARGET}" \
-                {{ $db.name | quote }}
+              BACKUP_DATE=$(date --utc +%Y-%m-%d_%H%M)
+              DATABASE_NAME={{ $db.name | quote }}
+              CURRENT_TRIPLE_COUNT=$(/opt/stardog/bin/stardog data size --exact "${DATABASE_NAME}" --passwd "${STARDOG_ADMIN_PW}" | cut -d " " -f 2 | tr -d ",")
+              echo "${DATABASE_NAME} has ${CURRENT_TRIPLE_COUNT} triples."
+              BACKUP_TARGET="s3://${S3_ENDPOINT}/${S3_BUCKET_NAME}/${S3_PATH_PREFIX}/${BACKUP_DATE}?AWS_ACCESS_KEY_ID=${S3_ACCESS_KEY}&AWS_SECRET_ACCESS_KEY=${S3_SECRET_KEY}&region=${S3_REGION}"
+              /opt/stardog/bin/stardog-admin db status --passwd "${STARDOG_ADMIN_PW}" "${DATABASE_NAME}"
+              /opt/stardog/bin/stardog-admin db backup --passwd "${STARDOG_ADMIN_PW}" --to "${BACKUP_TARGET}" "${DATABASE_NAME}"
+              PERCENT_USE=$(df -h | grep var.opt.stardog | tr -s " " | cut -d ' ' -f 5 | tr -d "%")
```

It's probably easier to specify the target for the `df` command and you can use `--output=pcent` to only show the percentage:
```suggestion
              PERCENT_USE=$(df --output=pcent -h /var/opt/stardog | tail -n 1 | tr -d "%")
```

---

Yes, that's cleaner.